data/compatible_products.yml

```diff
@@ -30,10 +30,10 @@ sets:
       - name: Payara Server 
         vendor: Payara Services Limited
         image: '/images/compatible_products/payara_server.png'
-        download: 'https://www.payara.fish/software/downloads/'
+        download: 'https://www.payara.fish/downloads/'
         versions:
-        - version: 5.21.2
```

Comment:
Did they ask us to remove 5.21.2? I was under the impression that we would be expected to add to the list instead of replace as we did for https://github.com/jakartaee/jakarta.ee/blob/be4c319dc42d92fcceaebf7ab9a114df53491284/data/compatible_products.yml#L123

Reply:
@flora8984461 Can you confirm the changes in the issue?

Reply:
waiting for confirming https://github.com/jakartaee/jakarta.ee/issues/1005#issuecomment-735944154